docs/pages/guides/using-eslint.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Use ESLint
+description: How to configure ESLint for formatting Expo apps.
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+import { FileTree } from '~/ui/components/FileTree';
+import { Tabs, Tab, TabsGroup } from '~/ui/components/Tabs';
+import { Step } from '~/ui/components/Step';
+
+<TabsGroup>
+
+[ESLint](https://eslint.org/) is a JavaScript linter that helps you find and fix errors in your code. It's a great tool to help you write better code and catch mistakes before they make it to production.
+
+## Setup
+
+<Step label="1">
+
+Install ESLint in your project.
+
+<Tabs>
+    <Tab label="npm">
+        <Terminal cmd={['$ npm install eslint eslint-config-universe --save-dev']} />
+    </Tab>
+
+    <Tab label="yarn">
+        <Terminal cmd={['$ yarn add -D eslint eslint-config-universe']} />
+    </Tab>
+
+</Tabs>
+
+</Step>
+
+<Step label="2">
+
+Create an ESLint config file––**.eslintrc.js**––in the project root.
+
+```js .eslintrc.js
+module.exports = {
+  root: true,
+  extends: [
+    /* @info This is the internal preset used by the Expo team. You can use any plugin you'd like. */
+    'universe/native',
+    /* @end */
+  ],
+};
+```
+
+</Step>
+
+<Step label="3">
+
+Add a **script** to your **package.json** to run ESLint.
+
+```json package.json
+{
+  "scripts": {
+    "lint": "eslint ."
+  }
+}
+```
+
+You may want to replace the `.` with specific directories or files to lint. For example, if you use Expo Router, you can use `eslint app` to lint only your routes.
+
+</Step>
+
+## Usage
+
+You can lint your code manually from the command-line with with the script:
+
+<Tabs>
+    <Tab label="npm">
+        <Terminal cmd={['$ npm run lint']} />
+    </Tab>
+
+    <Tab label="yarn">
+        <Terminal cmd={['$ yarn lint']} />
+    </Tab>
+
+</Tabs>
+
+If you're using VS Code, you can install the [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) to lint your code as you type (recommended).
+
+## Envrionment configuration
+
+ESLint is generally configured for a single environment, the source code in an Expo app, while written in JavaScript, is run in multiple different environments. For example, the **app.config.js**, **metro.config.js**, **babel.config.js**, and **app/+html.tsx** files are run in Node.js envrionments, this means they have access to the global `__dirname` variable, and can use Node.js modules like `path`. Standard Expo project files like **app/index.js** can be run in Hermes, Node.js, or the web browser.
```

Comment:
```suggestion
ESLint is generally configured for a single environment. However, the source code is written in JavaScript in an Expo app runs in multiple different environments. For example, the **app.config.js**, **metro.config.js**, **babel.config.js**, and **app/+html.tsx** files are run in a Node.js environment. It means they have access to the global `__dirname` variable and can use Node.js modules such as `path`. Standard Expo project files like **app/index.js** can be run in Hermes, Node.js, or the web browser.
```

- The first sentence has too many pauses, so created separation by splitting it into two sentences.
- Fixed grammatical errors